File: ExampleApp/ExampleApp.cpp
#include "MainWindow.h"

#include <QtWidgets/qapplication.h>

static HINSTANCE appInstance;

namespace ExampleApp
{
   /////////////////////////////////////////////////////////////////////////
   //
   // Creates the Qt App and a main window.

   int App(int argc, char** argv)
   {
      QScopedPointer<QApplication> app(new QApplication(argc, argv));

      auto mainWindow = new MainWindow;

      mainWindow->resize(1000, 800);
      mainWindow->show();

      return app->exec();
   }
}

int main(int argc, char** argv)
{
   return ExampleApp::App(argc, argv);
}

int wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, wchar_t* lpCmdLine, int nCmdShow)
{
   appInstance = hInstance;
   return main(0, 0);
}

File: PythonExample/exampleApp.py
from PySide2.QtWidgets import QApplication
from mainWindow import MainWindow
import sys

def main():
    app = QApplication(sys.argv)
    main_window = MainWindow()
    #main_window.resize(1000,800)
    main_window.show()
    return app.exec_()
    

if __name__ == '__main__':
    sys.exit(main())
    
